### PR TITLE
Create SpaceShuttleEngines-2.1.3.ckan

### PR DESCRIPTION
**Pull request by PickledTripod**
I'd contribute in NetKAN, but the mod isn't on Github, KerbalStuff or anything readable by it as far as I know. It also won't be updated anytime soon since dtobi has retired.

### CODE REVIEW

**Comment by hakan42** [comment]
Could you please change the license to CC-BY-NC-ND-3.0 so the validator is happy?

Also, you could write up multiple authors as:

```
"author": [ "dtobi", "sarbian" ],
```

File: SpaceShuttleEngines/SpaceShuttleEngines-2.1.3.ckan [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+    "spec_version"  : 1,
+    "identifier"    : "SpaceShuttleEngines",
+    "name"          : "Space Shuttle Engines",
+    "abstract"      : "Shuttle engines with a high gimbal range, OMS engines, external tanks and more."
+    "version"       : "2.1.3",
+    "author"        : "dtobi, Sarbian"
+    "license"       : "CC BY-NC-ND 3.0",
```
Comment (PickledTripod) [inline SpaceShuttleEngines/SpaceShuttleEngines-2.1.3.ckan]: hjv.k.,jhjk.

**Comment by PickledTripod** [comment]
How do I edit it again? Apparently I don't have access to the branch I created...
Sorry, I'm new to github.

**Comment by hakan42** [comment]
Ah, you created this branch with the web interface, right? Let me merge it as it is and then I'll repair it...

**Comment by pjf** [comment]
Through the powers of Raise Thread and Gitomancy, I've merged this with a few metadata syntax corrections.

Many thanks!

~ pjf

```
$ ckan.exe show SpaceShuttleEngines
Space Shuttle Engines: Shuttle engines with a high gimbal range, OMS engines, external tanks and more.

Module info:
- version:	2.1.3
- authors:	dtobi, Sarbian
- status:	stable
- license:	CC-BY-NC-ND-3.0

Depends:
- KlockheedMartian-Gimbal

Suggests:
- SmokeScreen

Provides:
- SpaceShuttleEngines

Resources:
- homepage: http://forum.kerbalspaceprogram.com/threads/55985

Showing 154 installed files:
- GameData/Klockheed_Martian_SSE
- GameData/Klockheed_Martian_SSE/FX
- GameData/Klockheed_Martian_SSE/FX/cryo-1m
- GameData/Klockheed_Martian_SSE/FX/cryo-1m/cloud.tga
- GameData/Klockheed_Martian_SSE/FX/cryo-1m/model.mu
- GameData/Klockheed_Martian_SSE/FX/engine
- GameData/Klockheed_Martian_SSE/FX/engine/airflame.mu
- GameData/Klockheed_Martian_SSE/FX/engine/airflamel.mu
- GameData/Klockheed_Martian_SSE/FX/engine/airflamer.mu
- GameData/Klockheed_Martian_SSE/FX/engine/bbflame.mu
- GameData/Klockheed_Martian_SSE/FX/engine/cloud.tga
- GameData/Klockheed_Martian_SSE/FX/engine/engineFXaero.mu
- GameData/Klockheed_Martian_SSE/FX/engine/oms-fx.mu
- GameData/Klockheed_Martian_SSE/FX/engine/omsflame.mu
- GameData/Klockheed_Martian_SSE/FX/engine/ssmeeffect-red.tga
- GameData/Klockheed_Martian_SSE/FX/engine/ssmeeffect-red2.tga
- GameData/Klockheed_Martian_SSE/FX/engine/ssmeeffect.tga
- GameData/Klockheed_Martian_SSE/FX/engine/ssmeflame.mu
- GameData/Klockheed_Martian_SSE/FX/engine/x0flame.mu
- GameData/Klockheed_Martian_SSE/FX/engine/x0OMSflame.mu
- GameData/Klockheed_Martian_SSE/FX/engine/x2flame.mu
- GameData/Klockheed_Martian_SSE/Parts
- GameData/Klockheed_Martian_SSE/Parts/Bigbang1
- GameData/Klockheed_Martian_SSE/Parts/Bigbang1/bb-complete2.tga
- GameData/Klockheed_Martian_SSE/Parts/Bigbang1/bb-HEAT 1.tga
- GameData/Klockheed_Martian_SSE/Parts/Bigbang1/bb1.mu
- GameData/Klockheed_Martian_SSE/Parts/Bigbang1/cloud.tga
- GameData/Klockheed_Martian_SSE/Parts/Bigbang1/part.cfg
- GameData/Klockheed_Martian_SSE/Parts/Bigbang2
- GameData/Klockheed_Martian_SSE/Parts/Bigbang2/bb-complete-grey.tga
- GameData/Klockheed_Martian_SSE/Parts/Bigbang2/bb-HEAT 1.tga
- GameData/Klockheed_Martian_SSE/Parts/Bigbang2/bb2.mu
- GameData/Klockheed_Martian_SSE/Parts/Bigbang2/cloud.tga
- GameData/Klockheed_Martian_SSE/Parts/Bigbang2/part.cfg
- GameData/Klockheed_Martian_SSE/Parts/Double-oms
- GameData/Klockheed_Martian_SSE/Parts/Double-oms/double-oms.mu
- GameData/Klockheed_Martian_SSE/Parts/Double-oms/dualomv7-alpha.tga
- GameData/Klockheed_Martian_SSE/Parts/Double-oms/dualomv7-heat.tga
- GameData/Klockheed_Martian_SSE/Parts/Double-oms/part.cfg
- GameData/Klockheed_Martian_SSE/Parts/Multi Adapter
- GameData/Klockheed_Martian_SSE/Parts/Multi Adapter/ma0.cfg
- GameData/Klockheed_Martian_SSE/Parts/Multi Adapter/ma1.cfg
- GameData/Klockheed_Martian_SSE/Parts/Multi Adapter/model.mu
- GameData/Klockheed_Martian_SSE/Parts/Multi Adapter/multi-adapter-complete.tga
- GameData/Klockheed_Martian_SSE/Parts/Oms
- GameData/Klockheed_Martian_SSE/Parts/Oms/dualomv7-alpha.tga
- GameData/Klockheed_Martian_SSE/Parts/Oms/dualomv7-heat 1.tga
- GameData/Klockheed_Martian_SSE/Parts/Oms/dualomv7-heat.tga
- GameData/Klockheed_Martian_SSE/Parts/Oms/oms.mu
- GameData/Klockheed_Martian_SSE/Parts/Oms/part.cfg
- GameData/Klockheed_Martian_SSE/Parts/Shoulders
- GameData/Klockheed_Martian_SSE/Parts/Shoulders/part.cfg
- GameData/Klockheed_Martian_SSE/Parts/Shoulders/sh1.mu
- GameData/Klockheed_Martian_SSE/Parts/Shoulders/shoulders3-complete 2.tga
- GameData/Klockheed_Martian_SSE/Parts/Shuttle Engine X0
- GameData/Klockheed_Martian_SSE/Parts/Shuttle Engine X0-OMS
- GameData/Klockheed_Martian_SSE/Parts/Shuttle Engine X0-OMS/cloud.tga
- GameData/Klockheed_Martian_SSE/Parts/Shuttle Engine X0-OMS/fairing-complete.tga
- GameData/Klockheed_Martian_SSE/Parts/Shuttle Engine X0-OMS/part.cfg
- GameData/Klockheed_Martian_SSE/Parts/Shuttle Engine X0-OMS/x0-alpha-heat.tga
- GameData/Klockheed_Martian_SSE/Parts/Shuttle Engine X0-OMS/x0-complete.tga
- GameData/Klockheed_Martian_SSE/Parts/Shuttle Engine X0-OMS/x0-oms.mu
- GameData/Klockheed_Martian_SSE/Parts/Shuttle Engine X0/cloud.tga
- GameData/Klockheed_Martian_SSE/Parts/Shuttle Engine X0/part.cfg
- GameData/Klockheed_Martian_SSE/Parts/Shuttle Engine X0/x0-alpha-heat.tga
- GameData/Klockheed_Martian_SSE/Parts/Shuttle Engine X0/x0-complete.tga
- GameData/Klockheed_Martian_SSE/Parts/Shuttle Engine X0/x0.mu
- GameData/Klockheed_Martian_SSE/Parts/Shuttle Engine X2
- GameData/Klockheed_Martian_SSE/Parts/Shuttle Engine X2/cloud.tga
- GameData/Klockheed_Martian_SSE/Parts/Shuttle Engine X2/nozzle8-heat 3.tga
- GameData/Klockheed_Martian_SSE/Parts/Shuttle Engine X2/nozzle8.tga
- GameData/Klockheed_Martian_SSE/Parts/Shuttle Engine X2/part.cfg
- GameData/Klockheed_Martian_SSE/Parts/Shuttle Engine X2/se2 - Copy.mux
- GameData/Klockheed_Martian_SSE/Parts/Shuttle Engine X2/se2.mu
- GameData/Klockheed_Martian_SSE/Parts/Shuttle Engine X3
- GameData/Klockheed_Martian_SSE/Parts/Shuttle Engine X3/cloud.tga
- GameData/Klockheed_Martian_SSE/Parts/Shuttle Engine X3/part.cfg
- GameData/Klockheed_Martian_SSE/Parts/Shuttle Engine X3/x3-complete.tga
- GameData/Klockheed_Martian_SSE/Parts/Shuttle Engine X3/x3-heat.tga
- GameData/Klockheed_Martian_SSE/Parts/Shuttle Engine X3/x3.mu
- GameData/Klockheed_Martian_SSE/Parts/Shuttle Engine X4
- GameData/Klockheed_Martian_SSE/Parts/Shuttle Engine X4/cloud.tga
- GameData/Klockheed_Martian_SSE/Parts/Shuttle Engine X4/part2.cfg
- GameData/Klockheed_Martian_SSE/Parts/Shuttle Engine X4/x4-alpha-heat.tga
- GameData/Klockheed_Martian_SSE/Parts/Shuttle Engine X4/x4-complete.tga
- GameData/Klockheed_Martian_SSE/Parts/Shuttle Engine X4/x4l.mu
- GameData/Klockheed_Martian_SSE/Parts/Smallshoulders
- GameData/Klockheed_Martian_SSE/Parts/Smallshoulders/Sh0.cfg
- GameData/Klockheed_Martian_SSE/Parts/Smallshoulders/Sh1.cfg
- GameData/Klockheed_Martian_SSE/Parts/Smallshoulders/smallshoulders-complete.tga
- GameData/Klockheed_Martian_SSE/Parts/Smallshoulders/smallshoulders.mu
- GameData/Klockheed_Martian_SSE/Parts/Sparkle
- GameData/Klockheed_Martian_SSE/Parts/Sparkle/softlanding-complete.tga
- GameData/Klockheed_Martian_SSE/Parts/Sparkle/sparkle.cfg
- GameData/Klockheed_Martian_SSE/Parts/Sparkle/sparkle.mu
- GameData/Klockheed_Martian_SSE/Parts/Sparkle/sparkle.tga
- GameData/Klockheed_Martian_SSE/Parts/Ssme-s
- GameData/Klockheed_Martian_SSE/Parts/Ssme-s-Straight
- GameData/Klockheed_Martian_SSE/Parts/Ssme-s-Straight/cloud.tga
- GameData/Klockheed_Martian_SSE/Parts/Ssme-s-Straight/fairing.tga
- GameData/Klockheed_Martian_SSE/Parts/Ssme-s-Straight/model.mu
- GameData/Klockheed_Martian_SSE/Parts/Ssme-s-Straight/part-straight.cfg
- GameData/Klockheed_Martian_SSE/Parts/Ssme-s-Straight/ssme-map-ALPHA-COMPLETE.tga
- GameData/Klockheed_Martian_SSE/Parts/Ssme-s-Straight/ssme-map-bump_NRM.tga
- GameData/Klockheed_Martian_SSE/Parts/Ssme-s-Straight/ssme-map-heat.tga
- GameData/Klockheed_Martian_SSE/Parts/Ssme-s-Straight/ssme-map.tga
- GameData/Klockheed_Martian_SSE/Parts/Ssme-s-Straight/ssmeeffect.tga
- GameData/Klockheed_Martian_SSE/Parts/Ssme-s/cloud.tga
- GameData/Klockheed_Martian_SSE/Parts/Ssme-s/model.mu
- GameData/Klockheed_Martian_SSE/Parts/Ssme-s/part.cfg
- GameData/Klockheed_Martian_SSE/Parts/Ssme-s/ssme-map-ALPHA-COMPLETE.tga
- GameData/Klockheed_Martian_SSE/Parts/Ssme-s/ssme-map-bump_NRM.tga
- GameData/Klockheed_Martian_SSE/Parts/Ssme-s/ssme-map-heat.tga
- GameData/Klockheed_Martian_SSE/Parts/Ssme-s/ssme-map.tga
- GameData/Klockheed_Martian_SSE/Parts/Ssme-s/ssmeeffect.tga
- GameData/Klockheed_Martian_SSE/Parts/Ssme-s/x4-alpha-heat.tga
- GameData/Klockheed_Martian_SSE/Parts/Ssme-s/x4-complete.tga
- GameData/Klockheed_Martian_SSE/Parts/Tanks
- GameData/Klockheed_Martian_SSE/Parts/Tanks/bottom
- GameData/Klockheed_Martian_SSE/Parts/Tanks/bottom/B1.cfg
- GameData/Klockheed_Martian_SSE/Parts/Tanks/bottom/B2.cfg
- GameData/Klockheed_Martian_SSE/Parts/Tanks/bottom/B3.cfg
- GameData/Klockheed_Martian_SSE/Parts/Tanks/bottom/B4.cfg
- GameData/Klockheed_Martian_SSE/Parts/Tanks/bottom/km-tank-bottom-2.mu
- GameData/Klockheed_Martian_SSE/Parts/Tanks/bottom/orange-top-bump_NRM.tga
- GameData/Klockheed_Martian_SSE/Parts/Tanks/bottom/orange-top-COMPLETE.tga
- GameData/Klockheed_Martian_SSE/Parts/Tanks/bottom/orange-top.tga
- GameData/Klockheed_Martian_SSE/Parts/Tanks/mid
- GameData/Klockheed_Martian_SSE/Parts/Tanks/mid-small
- GameData/Klockheed_Martian_SSE/Parts/Tanks/mid-small/km-tank-mid-small-2.mu
- GameData/Klockheed_Martian_SSE/Parts/Tanks/mid-small/orange-mid-large-complete.tga
- GameData/Klockheed_Martian_SSE/Parts/Tanks/mid-small/orange-top-bump_NRM.tga
- GameData/Klockheed_Martian_SSE/Parts/Tanks/mid-small/orange-top-COMPLETE.tga
- GameData/Klockheed_Martian_SSE/Parts/Tanks/mid-small/S1.cfg
- GameData/Klockheed_Martian_SSE/Parts/Tanks/mid-small/S2.cfg
- GameData/Klockheed_Martian_SSE/Parts/Tanks/mid-small/S3.cfg
- GameData/Klockheed_Martian_SSE/Parts/Tanks/mid-small/S4.cfg
- GameData/Klockheed_Martian_SSE/Parts/Tanks/mid/km-tank-mid-2.mu
- GameData/Klockheed_Martian_SSE/Parts/Tanks/mid/M1.cfg
- GameData/Klockheed_Martian_SSE/Parts/Tanks/mid/M2.cfg
- GameData/Klockheed_Martian_SSE/Parts/Tanks/mid/M3.cfg
- GameData/Klockheed_Martian_SSE/Parts/Tanks/mid/M4.cfg
- GameData/Klockheed_Martian_SSE/Parts/Tanks/mid/orange-mid-large-complete.tga
- GameData/Klockheed_Martian_SSE/Parts/Tanks/top
- GameData/Klockheed_Martian_SSE/Parts/Tanks/top/cloud.tga
- GameData/Klockheed_Martian_SSE/Parts/Tanks/top/km-tank-top-2.mu
- GameData/Klockheed_Martian_SSE/Parts/Tanks/top/orange-top-bump_NRM.tga
- GameData/Klockheed_Martian_SSE/Parts/Tanks/top/orange-top-COMPLETE.tga
- GameData/Klockheed_Martian_SSE/Parts/Tanks/top/orange-top.tga
- GameData/Klockheed_Martian_SSE/Parts/Tanks/top/T1.cfg
- GameData/Klockheed_Martian_SSE/Parts/Tanks/top/T2.cfg
- GameData/Klockheed_Martian_SSE/Parts/Tanks/top/T3.cfg
- GameData/Klockheed_Martian_SSE/Parts/Tanks/top/T4.cfg
- GameData/Klockheed_Martian_SSE/Plugins
```

**Pull request closed without merging**